docs/wallets/ledger-wallet.mdx

```diff
@@ -19,4 +19,215 @@ for:
 - Transferring tokens
 - Signing transactions
 
+:::important
+
 At this time, Hotspot management actions such as onboarding or asserting location are not supported.
+
+:::
+
+## How To Access Ledger Account in Helium Mobile Wallet
+
+So let's take a look at how to access your Ledger account via the Helium Mobile Wallet app.
+
+### Step 1: Preparation
+Before you begin, there are prerequsites and software you need to install which includes:
+
+* Helium Ledger Nano X Device
+* [Ledger Live Application](https://www.ledger.com/ledger-live)
+* Helium Mobile Wallet app
+
+Nano X users need to have Bluetooth connectivity to manage HNT using the app.
+
+### Step 2: Install the Solana app on your Ledger device
+
+:::important
+
+Helium migration to Solana
+
+Helium blockchain migrated to Solana blockchain. The migration to Solana took place on April 18, 2023, beginning at 9am PST. After the migration the Helium blockchain is officially halted. Learn more
```

Comment:
there's no link here on learn more.

Also why do we need to say in this doc that the blockchain is halted?

Reply:
I deleted this part.